playbooks/ansible-setup.yml

```diff
@@ -2,14 +2,42 @@
 - hosts: all
   gather_facts: no
   tasks:
-  - name: Install dependencies
-    raw: dnf install -y python python-dnf
+  - name: setup
```

Comment:
'setup' task will fail on fedora cloud image if python is not installed.
'package' task will fail on fedora cloud image if python-dnf is not installed.
Maybe tag the old tasks with the RedHat condition, and the new tasks with Debian condition?